zcash_client_backend/src/data_api/wallet/input_selection.rs

```diff
@@ -21,12 +20,18 @@ use zcash_primitives::{
 
 use crate::{
     address::{RecipientAddress, UnifiedAddress},
-    data_api::WalletRead,
```

Comment:
The ```[`WalletRead`]``` intra-doc link on `GreedyInputSelector` breaks with this removal. Add a manual link to its documentation (or change the documentation accordingly).